_projects/meta_review.md

```diff
@@ -0,0 +1,95 @@
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - "https://github.com/coala/projects/issues/480"
+markdown: meta_review.md
+mentors:
+  - hemangsk
+  - blazeu
+  - alisianoi
+  - andrewda
+name: "Meta-review"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "The applicant should be familiar with GitHub GraphQL API and JavaScript."
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The meta-review project is about building a meta-review system,
+similar to a meta-moderation system, that can track emoji responses to
+review comments. These responses are to be collected, processed and displayed
+on [gh-board][https://github.com/coala/gh-board], which is a nice serverless
+kanban board.
+
+People including the author of pull request respond to comments by attaching
+emojis. Those emojis are called meta-review, or
+[reactions][https://developer.github.com/v4/enum/reactioncontent/]. THUMBS_UP
+and THUMBS_DOWN, together with other reactions, are used by people especially
+the author of pull request to publicly acknowledge or reject review comments.
+This provides feedback to reviewers and all other members.
+
+However, those reactions spread over all PRs and are not collected and
+analyzed. By tracking reactions, a nice feedback loop would be created
+for both senior and junior reviewers. Also, this information can be used
+as newcomer metrics.
+
+One goal of this project is to encourage people to do meta-reviews,
+systematically. Worst case is that the tracking stores in the json which
+PR do not have meta-reviews, and gh-board shows only a list of those PRs
+(and the user must then find the reviews in GitHub PR webapp, and do
+meta-review there). An improvement is to store the identifier of the PR
+comments which do not have meta-reviews, and gh-board links to the PR
+review comment in GitHub PR webapp. In the ultimate version, gh-board
+would show the review comments.
+
+Also, to encourage people to do more meta-reviews, statistics of meta-reviews
+are to be collected and analysed. People who do meta-reviews will get scores
+according to some metrics. A ranking list is to be displayed on gh-board.
+
+The scope of this project would exclude non-public meta-reviews. That is,
+there would not be any anonymous meta-reviews as that introduces too many
+additional complexities. The public only version would be used for
+long time before adding support for non-public meta-reviews.
+
+#### Milestones
+
+##### PREPARATION/BONDING
+
+- The applicant has proposed and merged a cEP of how meta-reviews could be
+  tracked and handled.
+- The applicant has created mockups for the frontend.
+- The applicant gets acquainted with materialize CSS.
+- The applicant gets acquainted with data visualization
+  library (e.g. d3.js).
+- The applicant gets
+  [gh-board#13][https://github.com/coala/gh-board/issues/13] fixed.
+- The applicant refactors React code a bit.
```

Comment:
I've been thinking, maybe "Modernize existing React component code in gh-board project" makes it more clear here.